ProgressBar/example.jsx

```diff
@@ -2,6 +2,28 @@ import React from 'react'
 import ProgressBar from '../ProgressBar'
 import { LIVE } from '../Showroom/variationTypes'
 
+class TimedExample extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      percentage: 0
+    }
+  }
+
+  componentDidMount () {
+    setInterval(() => {
```

Comment:
Unbind this in `componentWillUnmount`. No need to trick people into adding memory leaks.